isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -783,6 +784,785 @@ namespace Isis {
   }
 
 
+
+  /**
+   * @brief Takes in an open std::ofstream and writes out information which goes into the
+   * bundleout.txt file.
+   * @param fpOut The open std::ofstream object which is passed in from
+   * BundleSolutionInfo::outputText()
+   * @param errorPropagation Boolean indicating whether or not to attach more information
+   *     (corrections, sigmas, adjusted sigmas...) to the output.
+   */
+
+  void BundleObservation::bundleOutput(std::ofstream &fpOut, bool errorPropagation) {
```

Comment:
Wait - why do both this method and BundleObservation::bundleOutputString exist? They both have the same logic...